request-template-with-oauth/config/requests.json

```diff
@@ -0,0 +1,16 @@
+{
+    "asanaGetWorkspace": {
+        "schema": {
+            "method": "GET",
+            "host": "app.asana.com",
+            "path": "/api/1.0/workspaces",
+            "headers": {
+                "Authorization": "bearer <%=access_token%>",
```

Comment:
Can you change this for uniformity:

`"Bearer <%= access_token %>"`

Reply:
Updated.